starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/dependencies.rocker.raw

```diff
@@ -492,13 +492,13 @@ dependencies {
 @if (features.contains("google-cloud-function") && features.testFramework().isSpock()) {
     @dependency.template("io.micronaut.servlet","micronaut-servlet-core", "testImplementation", null)
 }
-@if (features.contains("mongo-reactive") || features.contains("mongo-sync")) {
-    @dependency.template("de.flapdoodle.embed", "de.flapdoodle.embed.mongo", "testImplementation", "2.0.1")
-}
 @if (features.contains("neo4j-bolt")) {
     @dependency.template("org.neo4j.test", "neo4j-harness", "testRuntimeOnly", null)
 }
 @if (features.contains("testcontainers")) {
+@if (features.contains("mongo-reactive") || features.contains("mongo-sync")) {
+    @dependency.template("org.testcontainers", "mongodb", "testImplementation", null)
+}
```

Comment:
Question: if `mongo` is now nested inside `testcontainers`, is it possible to select only mongo without selected testcontainers?

Reply:
No, if a mongodb feature is selected, you get the `testcontainers` dependencies for it. It works like that already for the other databases that use `testcontainers` (MySql, Oracle, MariaDb, etc). For each of them, if you select the database feature, but not the `testcontainers` feature, then `testcontainers` dependencies (and now junit/spock annotations for it) are included automatically. 

OTOH, if the `testcontainers` feature is selected, then the `application-test.yml` file with the "tc" driver url is generated. However, mongodb `testcontainers` doesn't use the special "tc" driver url, so it ends up making no difference whether one selects `testcontainers` or not. Given that, the logic is consistent for selecting just the database feature for mongodb as it is for the others.

Maybe `testcontainers` should just be an automatic feature for the databases?

Reply:
Sounds good. Thanks for the clarification.